mkosi/backend.py

```diff
@@ -112,6 +112,24 @@ class PackageType(enum.Enum):
     ebuild = 5
 
 
+class Verb(enum.Enum):
+    build   = "build"
+    clean   = "clean"
+    summary = "summary"
+    shell   = "shell"
+    boot    = "boot"
+    qemu    = "qemu"
+    ssh     = "ssh"
+    serve   = "serve"
+    bump    = "bump"
+    help    = "help"
+    genkey  = "genkey"
```

Comment:
I used tabbing to separate the verbs and '=' since this was done as well into other Enum classes with strings, such as `PartitionIdentifier` and `ManifestFormat`